scripts/all/build-it.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+SYSTEM=$1
```

Comment:
Could you put some validation or help text on these arguments? RIght now you can put in anything, or nothing, and it will run and give some slightly odd results.

Reply:
yes, I'll make it check for what we expect.